configure

```diff
@@ -23458,9 +23458,18 @@ if test ${enable_video_x11+y}
 then :
   enableval=$enable_video_x11;
 else $as_nop
-  enable_video_x11=yes
+  enable_video_x11=maybe
```

Comment:
This is forcefully disabling x11 for mac: the previous version was better IMO.
We should fix the egl+x11 mess, instead of force-disabling it for mac.

Reply:
I changed the logic.
You're right. Once #6348 is fixed, this commit can be dropped.

Reply:
It need not be dropped, but can be reverted to your previous logic which isn't actually bad.

Reply:
I kind of like the current approach: use the 4th argument of [`AC_ARG_ENABLE`](https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Package-Options.html) to determine the default.